src/sqsd.groovy

```diff
@@ -119,5 +119,5 @@ def handleMessage(String httpHost, String httpPath, String contentType, Message
 
     println "POST " + httpHost + httpPath + " :: " + status
 
-    status < 400
+    status == 200
```

Comment:
Good point, but I think it should it be `status < 300`. @ortimanu what do you think?